src/components/pipeline/CreatePipeline.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+import Main from '../common/Main';
+import BackButton from '../common/BackButton';
+import { EDIT_PIPELINE_BACK_BUTTON_ID } from '../../config/selectors';
+
+import PipelineForm from './PipelineForm';
+
+const styles = (theme) => ({
+  backButton: {
+    position: 'fixed',
+    right: theme.spacing(2),
+    bottom: theme.spacing(2),
+  },
+  content: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+  },
+});
+
+const CreatePipeline = (props) => {
+  const { t, classes } = props;
+
+  return (
+    <Main>
+      <Container className={classes.content}>
+        <Typography variant="h4">{t('Create Pipeline')}</Typography>
+        <PipelineForm pipelineAlgorithms={[]} name="" description="" />
+      </Container>
+      <BackButton
+        className={classes.backButton}
+        id={EDIT_PIPELINE_BACK_BUTTON_ID}
+      />
+    </Main>
+  );
+};
+
+CreatePipeline.propTypes = {
+  t: PropTypes.func.isRequired,
+  classes: PropTypes.shape({
+    content: PropTypes.string.isRequired,
+    backButton: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+const StyledComponent = withStyles(styles)(CreatePipeline);
+
+const TranslatedComponent = withTranslation()(StyledComponent);
+
+export default withRouter(TranslatedComponent);
```

Comment:
I think you don't need to wrap this component with the router.

Reply:
`CreatePipeline`, `EditPipeline` and `Pipelines` are wrapped with `withRouter`. Those components are directly used in the `App.js` for routing. Therefore, I'm not really sure if I need to remove the `withRouter` wrapping for this specific component.